calicoctl/commands/get.go

```diff
@@ -65,9 +65,17 @@ Description:
   or by type and identifiers.  JSON and YAML formats are accepted for file and
   stdin format.
 
-  Valid resource types are node, bgpPeer, hostEndpoint, workloadEndpoint,
-  ipPool, policy and profile.  The <TYPE> is case insensitive and may be
-  pluralized.
+  Valid resource types are:
+
+		* node
```

Comment:
I think indentation of the list is wrong - shouldn't be tabs and shouldn't be indented so much!